lib/pages/my_games_page.dart

```diff
@@ -1,18 +1,78 @@
 import 'package:flutter/material.dart';
-import 'package:go_router/go_router.dart';
+import 'package:join_play/blocs/authentication/bloc/authentication_bloc.dart';
+import 'package:join_play/models/sport_event.dart';
+import 'package:join_play/utilities/firebase_service.dart';
 
-import '../navigation/route_names.dart';
+class MyGamesPage extends StatefulWidget {
+  final FirebaseService firebaseService;
+  final AuthenticationBloc authenticationBloc;
 
-class MyGamesPage extends StatelessWidget {
-  const MyGamesPage({super.key});
+  const MyGamesPage({
+    super.key,
+    required this.firebaseService,
+    required this.authenticationBloc,
+  });
 
+  @override
+  State<MyGamesPage> createState() => _MyGamesPageState();
+}
+
+class _MyGamesPageState extends State<MyGamesPage> {
   @override
   Widget build(BuildContext context) {
-    return Center(
-      child: FilledButton(
-        child: const Text("Go to History"),
-        onPressed: () {
-          context.goNamed(RouteNames.history);
+    final userId = widget.authenticationBloc.sportUser?.uuid ?? '';
+
+    return Scaffold(
```

Comment:
Do not use scaffold on the pages. The router is using the ScafoldWithNavBar as the shell of the rest of the app. That's why I removed them.

Reply:
done